projects/buddy-system.md

```diff
@@ -0,0 +1,98 @@
+# Buddy system
+
+## Goal
+The goal of the buddy system is multifaceted:
+
+* Ensure newcomers feel comfortable and part of our team
+* Speed up the process of gaining essential technical knowledge for working on Invenio
+* Develop buddies skills in communication and take larger responsibility in the project
+
+## Who is involved?
+As previously mentioned, the buddy system intends to improve newcomers' experience, but for doing so there are four main actors who have to take part:
+
+* The newcomer
+* Buddy
+* Newcomer's supervisor
+* Buddy system maintainer
+
+## Who is the Buddy?
+* A person assigned to a newcomer for the first two months to help in both technical and social aspects
+
+* Someone from another service team to ensure cross-team integration inside the section
+
+* Can not be the newcomer’s supervisor
+
+* The section leader is responsible of selecting the newcomer's buddy
+
+* Should be available during the first two weeks after the newcomer arrives, and should not have extended absences (i.e. >2-3 weeks) during the first two months
+
+* For absences of a week or more, the buddy should ensure to hand-over buddy responsibilities to a colleague
+
+## What are the responsibilities?
+
+In general buddy's responsibilities are:
+
+1. Being cool with her/his newcomer :sunglasses:
+2. Being available for her/him
+3. Arrange informal meetings with her/him as often as needed
+4. Follow-up with the buddy system maintainer
+5. Turn to the buddy system maintainer any time if needed
+
+Besides, since the process will span over a period of two months, a more specific list divided by phases will be provided.
+
+### Before newcomers arrival
+The buddy must meet the newcomer’s supervisor in order to get an overview of her/his project and discuss the first tasks:
+
+* Schedule when supervisor will introduce buddy to the newcomer
+* Define tasks on Invenio
+  * One of the following small tasks: read up relevant Invenio documentation, rehearse the presentations, test tutorials, check if any part of the material is outdated (~1 hr)
+  * A medium sized task (~1 day) suitable for pair-programming
+  * A first larger service task in which the supervisor will help
+* Agree on who is going to run the bootcamp following `inveniosoftware/training` repository.
+
+### First day
```

Comment:
Completely agree. Just about social events (like evening events) and official events (like Invenio Dev Forum), we leave this also to section leader and supervisor? Or perhaps creating a new section about it like social, or even adding them as a single line in the general duties (Something like, `Make sure that newcomer doesn't miss our social and official events`).